packages/react-native/src/theme/defaultTheme.ts

```diff
@@ -0,0 +1,7 @@
+import { tokens } from './tokens';
+import { Theme } from './types';
+
+export const defaultTheme: Theme = {
```

Comment:
Is this actually the default `Theme`? It doesn't contain the component default theme right?

Reply:
yes this is the default Theme

Reply:
Maybe I am misreading but the value of `tokens` here does not include the component tokens correct? So wouldn't that make this just the `baseTheme`?